tests/mgbench/datasets.py

```diff
@@ -103,8 +127,9 @@ class Pokec(Dataset):
     VARIANTS = ["small", "medium", "large"]
     DEFAULT_VARIANT = "small"
     FILES = None
+
     URLS = {
-        "small": "https://s3-eu-west-1.amazonaws.com/deps.memgraph.io/pokec_small.setup.cypher",
+        "small": "https://s3.eu-west-1.amazonaws.com/deps.memgraph.io/dataset/pokec/benchmark/pokec_small_import.cypher",
         "medium": "https://s3-eu-west-1.amazonaws.com/deps.memgraph.io/pokec_medium.setup.cypher",
         "large": "https://s3-eu-west-1.amazonaws.com/deps.memgraph.io/pokec_large.setup.cypher.gz",
```

Comment:
Adresses of medium and large should change as well

Reply:
https://s3.eu-west-1.amazonaws.com/deps.memgraph.io/dataset/pokec/pokec_small.setup.cypher
https://s3.eu-west-1.amazonaws.com/deps.memgraph.io/dataset/pokec/pokec_medium.setup.cypher
https://s3.eu-west-1.amazonaws.com/deps.memgraph.io/dataset/pokec/pokec_large.setup.cypher.gz

(from: https://github.com/memgraph/memgraph/blob/5c0e41ed44c283d172ecb9106bbc5809e8cb2906/tests/mgbench/datasets.py#L106-L110 )

Reply:
(maybe that's something we'll tackle when merging your PR to project-pineapple, so you can ignore this comment)